Tasks/Npm/task.json

```diff
@@ -32,6 +32,11 @@
             "displayName": "Destination registry and authentication",
             "isExpanded": true,
             "visibleRule": "command = publish"
+        },
+        {   "displayName": "Advanced Options",
+            "name": "advanced",
+            "isExpanded": false,
+            "visibleRule": "command = install || command = publish"
```

Comment:
Yes, in custom commands the user could type "--verbose" and check the box, which could lead to a situation where someone unchecks the verbose box (or deletes "--verbose" from their command), but verbose logging continues to happen. Keith Brian and Alex decided it would be best to not include it with custom commands.